eucaconsole/views/vpcs.py

```
@@ -326,9 +380,10 @@ def get_vpc_subnets(self):
             ))
         return subnets_list
 
-    def get_subnet_instances(self, subnet_id=None, vpc_reservations=None):
+    @staticmethod
+    def get_subnet_instances(subnet_id=None, vpc_reservations=None):
```

I guess I'd suggest that these should not default to invalid values, right? This function will return quickly if either is None, so why even have default?